gui/mozregui/wizard.py

```diff
@@ -111,6 +111,27 @@ def __init__(self):
         self.ui.start_date.setDateTime(now.addYears(-1))
         self.ui.end_date.setDateTime(now)
 
+    def validatePage(self):
+        good_date = self.ui.start_date.date()
+        bad_date = self.ui.end_date.date()
```

Comment:
Could you rename the variables here as "start_date" and "end_date" ? Because this code also handle the find fix feature (so good and bad will means the exact the opposite, which will be confusing) - but the start / end terminology will be good in any case. :)